package.json

```diff
@@ -23,7 +23,6 @@
     "chai": "^4.1.2",
     "eslint": "^4.1.1",
     "eslint-config-pedant": "^0.10.0",
-    "lerna": "^2.2.0",
```

Comment:
are you sure?
if you maintain several repos with different `lerna` versions wouldn't be any problem with this?